.config/.oxfmtrc.json

```diff
@@ -1,3 +1,10 @@
 {
-  "ignorePatterns": []
+  "rules": {
```

Comment:
I expect keeping all these rules enabled will just be annoying as it would be veeeery opinionated. What reasons do you see for keeping all categories enabled rather than selecting an individual set of rules from the more opinionated categories?